tests/Resources/ti.ui.navigationwindow.addontest.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2017-Present by Axway. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* global Ti */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+var should = require('./utilities/assertions');
```

Comment:
<!--
  0 failure: 
  1 warning:  tests/Resources/t...
  
  
  DangerID: danger-id-default;
  File: tests/Resources/ti.ui.navigationwindow.addontest.js;
  Line: 11;
-->

- :warning: tests/Resources/ti.ui.navigationwindow.addontest.js line 11 – 'should' is assigned a value but never used. (no-unused-vars)